### ISSUE TITLE
Updating IES Components

### ISSUE THREAD

**Issue by sariths** [mostaphaRoudsari/Honeybee#481]
@mostaphaRoudsari, @chriswmackey 

Based on some comments that I got from my colleagues, I am trying to add an export to Excel option in a new component. The Excel functionality is added by  the lines below:
```
import clr
clr.AddReference("Microsoft.Office.Interop.Excel")
import Microsoft.Office.Interop.Excel as Excel
```
Are you guys aware of (or foresee) any issues with trying to use an export to excel feature inside Grasshopper ?

PS: I have some other questions regarding updating existing components, that's why I am calling this issue as "updating ies components".

**Comment by mostaphaRoudsari** [comment]
@sariths do you really need excel or can you just export the data as csv file?

There are several plugins in Grasshopper (Lunchbox, TTToolBox, Bumble bee) which provide read and write feature to excel. If you do a quick search you can see different issues that can happen with excel libraries. I would try to use csv files if possible.

**Comment by sariths** [comment]
@mostaphaRoudsari Okay, case closed! I will stick to csv files. 
(I am keeping this issue open as I will be asking/checking a few other things on IES components in the coming days).

**Comment by sariths** [comment]
Turns out I didn't have a lot of questions. However, there is one which is more of a best practices kind of question...

The new component that I wrote consolidates all the luminaire rad files in one place: 
![2-16-2016 2-19-43 pm](https://cloud.githubusercontent.com/assets/10361848/13088098/e60c4864-d4b8-11e5-8b0f-ed930da219fa.png)

This is made possible by the new output called elecLightingData that I added to the IES Luminaire component. This improved workflow makes the existing radFilePath redundant. However, I have still left it on as the video and Hydra files etc use the radFilePath output.
![2-16-2016 2-24-33 pm](https://cloud.githubusercontent.com/assets/10361848/13088155/0e1b5764-d4b9-11e5-97ed-c78e2314c74c.png)

Bottom line is that right now both the new and old workflows are supported. Do you think I should remove the old workflow(and remove the radFilePath output) ?

**Comment by mostaphaRoudsari** [comment]
@sariths I think we should keep both of them. Let's just make sure we have a clear description for the input. It give a great a level of flexibility.

**Comment by sariths** [comment]
@mostaphaRoudsari Okay, noted. I will play around with these components for a while before I send you another PR with a better description/input comments. Thanks for acting on the current PR promptly.
